wagtail/utils/deprecation.py

```diff
@@ -2,5 +2,8 @@ class RemovedInWagtail12Warning(DeprecationWarning):
     pass
 
 
+removed_in_next_version_warning = RemovedInWagtail12Warning
```

Comment:
Further clarification: Up to Django 1.6, Django explicitly enabled all DeprecationWarnings. This was changed in Django 1.7 to only enable Django's own DeprecationWarning subclasses. This had the result of silencing Wagtail's DeprecationWarnings. Therefore, the correct course of action is for Wagtail to follow Django's lead and enable its own warnings, while still respecting the intent of the Django fix and leaving other DeprecationWarnings unchanged.

https://github.com/django/django/commit/0c6a3399523d850cfbd20de54cd089419a47383d
https://code.djangoproject.com/ticket/21188